### ISSUE TITLE
quickstarts/Function_calling.ipynb Can the ai only call functions once before returning the chat?

### ISSUE THREAD

**Issue by EnderRobber101** [google-gemini/cookbook#94]
Can the ai only call functions once before returning the chat?

**Comment by Gunand3043** [comment]
Yes, if you are talking about manual function calling, you can call it once, like an example [here](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb).

Thanks
